### PR TITLE
Updating to HLT paths to be monitored by TOP PAG in 2016 in 81X

### PR DESCRIPTION
**Pull request by soureek**
Removing Lepton + btags/jets triggers from the list of triggers as the TOP PAG does not intend to monitor them further in 2016. The reason being these triggers are not being used in TOP analyses. Also adding token for "hltTriggerSummaryAOD" collection present in both data and MC. Replaced "cout" commands with edm::LogWarning. The code is tested to be Ok by running the matrix for TTbar_13 workflow for 25ns bunch spacing.

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by dmitrijus** [comment]
80x: https://github.com/cms-sw/cmssw/pull/14367

File: DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py [hunk 1]
```diff
@@ -62,14 +62,17 @@
...
     triggerExtras = cms.PSet(
-        src = cms.InputTag("TriggerResults","","HLT"),
-        pathsELECMU = cms.vstring([ 'HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v', 
-                                    'HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v']),
-        pathsDIMUON = cms.vstring([ 'HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v',
-                                    'HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_DZ_v']),
-        pathsDIELEC = cms.vstring([ 'HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v', 
-                                    'HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_v'])
-    )    
+		src = cms.InputTag("TriggerResults","","HLT"),
```
Comment (dmitrijus) [inline DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py]: Please don't use tabs in python. Especially not if the rest of the file uses spaces.

**Comment by dmitrijus** [comment]
Please fix spaces/tabs in python files. In cpp compiler ignores whitespace, but in python this can lead to severe issues (since whitespace is used for blocks).

**Comment by Soureek89** [comment]
I am making changes to remove tabs and trying to commit them to the same branch.

**Comment by Soureek89** [comment]
Hello I have updated the branch by removing tabs according to the comments received. Can you please approve?

**Comment by perrotta** [comment]
please  test

**Comment by dmitrijus** [comment]
+1

**Comment by dmitrijus** [comment]
please test
There is a large, +200mb increase in memory used (I checked 25202.0).

**Comment by davidlange6** [comment]
+1

**Pull request merged**